marimbabot_msgs/action/HitSequence.action

```diff
@@ -15,3 +16,4 @@ uint16 error_code # error code in case of failure (see above)
 #feedback
 ---
 bool playing # whether marimbabot is currently (still) playing, i.e. the goal is still active
+marimbabot_msgs/HitSequenceElement[] executed_sequence_elements # see above
```

Comment:
Why is this named in past tense? When it is first published, they are not executed.